ci/build_wheel_libcuvs.sh

```diff
@@ -27,4 +27,28 @@ rapids-pip-retry install \
 export PIP_NO_BUILD_ISOLATION=0
 
 ci/build_wheel.sh libcuvs ${package_dir} cpp
-ci/validate_wheel.sh ${package_dir} "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"
+
+
+# We temporarily override the values in `libcuvs` `pyproject.toml` so we can
+# track the versions that have vendored NCCL (CUDA 11) vs. the ones that don't
+# (CUDA 12+)
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+WHEEL_SIZE_LIMIT='0.9G'
+if [[ ${RAPIDS_CUDA_MAJOR} == "11" ]]; then
+  WHEEL_SIZE_LIMIT='1.0G'
+fi
+
+
+rapids-logger "validate packages with 'pydistcheck'"
+cd "${package_dir}"
+
+pydistcheck \
+    --inspect \
+    --max-allowed-size-compressed $WHEEL_SIZE_LIMIT \
+    "$(echo "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"/*.whl)"
+
+rapids-logger "validate packages with 'twine'"
+
+twine check \
+    --strict \
+    "$(echo "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"/*.whl)"
```

Comment:
Would you consider instead putting this logic down in `validate_wheel.sh`, as we used to do for other projects?

See the diff at https://github.com/rapidsai/cugraph/pull/4804/files#diff-0565e15203407439a89a2d7c3da1ca9569b26bb583b039009ad85394f1ff70f9L9 for how we previously did this for `cugraph`, as an example.

I think that's preferable because doing that avoids duplicating the calls to `pydistcheck` and `twine check` across multiple places, keeping the "what after-building validation of wheels do we do?" logic in one place.

Reply:
yep, happy to make these changes!  I started along this path and got waylaid by overcomplicating it by attempting to handle passing bash arrays as arguments between files (tip, don't do this)